src/meta/client/MetaClient.h

```diff
@@ -153,7 +153,7 @@ class MetaClient {
     getSpace(std::string name);
 
     folly::Future<StatusOr<bool>>
-    dropSpace(std::string name);
+    dropSpace(std::string name, const bool ifExist = false);
```

Comment:
"const"  doesn't make much sense here.

Reply:
I think the argument no reason to be mutable.

Reply:
Yes, my mistake.